Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -129,4 +130,12 @@ RefPtr<Gfx::Bitmap> Layer::try_copy_bitmap(Selection const& selection) const
     return result;
 }
 
+void Layer::erase_selection(Selection const& selection)
+{
+    Gfx::Painter painter { bitmap() };
+    auto const image_and_selection_intersection = m_image.rect().intersected(selection.bounding_rect());
+    auto const translated_to_layer_space = image_and_selection_intersection.translated(-location());
+    painter.clear_rect(translated_to_layer_space, Color::Transparent);
+    did_modify_bitmap(translated_to_layer_space);
+}
```

Comment:
Please add a newline between the function's closing curly bracket and the namespace, as it was before.